### DISCUSSION TITLE
Duplicated cursor using dgVoodoo2

### DISCUSSION THREAD

**Discussion by Eliassims** [FaithBeam/Sims-1-Complete-Collection-Widescreen-Patcher#66] [General]
Hi ! is there a way to fix this issue ? it happens when i'm using the "Stretched, Keep Aspect Ratio" Option.  And I really like this option because is borderless. 

pd : I don't want to use DDrawcompat, I prefer dgVoodoo2

**Comment by voicemxil** [comment]
I noticed something similar when I tried dgvoodoo - the duplicate was in a smaller window in the top left corner
